README.md

```diff
@@ -47,6 +47,16 @@ ERROR: Issue doesn't exist.
 myproject/main.go:14: // TODO J321: A non-existent issue
 ```
 
+Here's another example of an annotated `TODO`:
+```
+// TODO #234: Fix the typo
+func fuu() {
+    ...
+}
+```
+
+In case the linked issue `234` with Github issue tracker is open, `todocheck` will not report any error. That means with Github issues, todos with a hashtag is supported.
+
```

Comment:
Sorry for seeing this just now, but this modification is not needed.

This section covers a high-level overview of how todocheck works. There is no need for a github-specific feature to be explained here.

Reply:
Should I also delete the example I added before?